ddtrace/contrib/grpc/client_interceptor.py

```diff
@@ -0,0 +1,54 @@
+import grpc
+
+from ddtrace import Pin
+from .propagation import inject_span
+
+class GrpcClientInterceptor(
+        grpc.UnaryUnaryClientInterceptor, grpc.UnaryStreamClientInterceptor,
+        grpc.StreamUnaryClientInterceptor, grpc.StreamStreamClientInterceptor):
+    """Intercept calls on a channel. It creates span as well as doing the propagation
+    Derived from https://github.com/grpc/grpc/blob/\
```

Comment:
Can we update this to have the link all on a single line so it is copy/pasteable?

The line will be too long, but you can add `  # noqa` (kind of hard to tell in this example, but two spaces, pound, space, "noqa") to the end of the line to disable the linting error